src/http/web_socket.cr

```diff
@@ -100,6 +100,20 @@ class HTTP::WebSocket
     @ws.close(message)
   end
 
+  # Will run the websocket in an infinite loop where it receives messages and calls previously set callbacks.
+  # The websocket will autmatically respond with pong on pings and exit the loop when the websocket is closed.
```

Comment:
```suggestion
  # Ping and pong messages are automatically handled.
```

It's not the socket that answers the request, but the loop inside this method :)

Reply:
Seems good to me!
(It's my first contribution so sorry for the small problems)

Reply:
Wanna apply the suggestions then? :) Since you didn't give maintainer push rights to your fork I can't do it myself in this PR :)